app/views/layouts/application.html.slim

```diff
@@ -1,6 +1,16 @@
 doctype html
 html
   head
+    - if ENV['GOOGLE_ANALYTICS_TRACKING_ID']
+      javascript:
+        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```

Comment:
did you copy and paste this function from somewhere? can you add a comment citing where you got it from and explaining what it's doing?

Reply:
Added comment for GA documentation of where I got the main script, the other piece is the `turbolinks:load` event which I found researching rails routing and it fires on each page change so I added a brief comment on that line as well